### ISSUE TITLE
Demo not working on iOS 14 emulator ?

### ISSUE THREAD

**Issue by emerabet** [SDWebImage/SDWebImageSwiftUI#120]
Hello,

I've just tested the example project on ios 14 emulator and a lot of images don't load, this is not the case on iOS 13.5.
How can I determine if this is coming from SDWebImageSwiftUI or Apple ?

Thanks !

**Comment by dreampiggy** [comment]
Let me try again.

Have you try to use `pod update` for that demo ?

SDWebImage fix a issue that `AnimatedIamge` not rendering on iOS 14.

**Comment by emerabet** [comment]
Thanks for your help :)

**Comment by dreampiggy** [comment]
Still have issues compatible for iOS 14.

The SVG/PDF vector images can not render on `AnimatedImage`.

This should be fixed via: https://github.com/SDWebImage/SDWebImage/pull/3046

I'll release SDWebImage 5.8.3 soon after that been merged.
